engine/physics/physics_world.cpp

```diff
@@ -18,11 +18,13 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
+//#include <cmath>
 
 #ifdef WIN32
 #include <float.h>
 #define isnan _isnan
```

Comment:
This will break in Windows because it will end up calling std::_isnan -- however you should be able to remove it as std::isnan will work in Windows.

While you're at it, you may as well remove the commented out #include <cmath> above.